lib/core/scene/scene-data-manager.ts

```diff
@@ -110,7 +127,13 @@ class SceneDataManager {
     private m_blasArray: Array<BLAS>;
     private m_blasNodeCount: number; // total node count
     private m_blasInstanceArray: Array<BLASInstance>;
+
     private m_materials: Array<Material>;
+    private m_totalMapCount: number;
+    private m_albedoMapCount: number;
```

Comment:
map counts are not used

Reply:
forgot to delete them. will delete them now.